noir-projects/noir-protocol-circuits/crates/rollup-lib/src/base/public_base_rollup.nr

```diff
@@ -1,29 +1,43 @@
 use crate::{
```

Comment:
Some code in the public base and public base tests is very similar to the code in private base. However the public base will get much more different now that we'll stop inserting in the trees in it, so it's been splitted in this PR